### PR TITLE
Migrate `<Popover>` to PF4

### PR DESCRIPTION
**Pull request by rhamilto**
Resolves https://jira.coreos.com/browse/CONSOLE-1720

Mock example
Screen Shot 2019-09-13 at 10 12 24 AM

### CODE REVIEW

**Comment by rhamilto** [comment]
/hold 

for 4.3

**Comment by rhamilto** [comment]
This should probably be updated to use https://github.com/openshift/console/pull/2733

File: frontend/public/vendor.scss [hunk 1]
```diff
@@ -40,7 +39,6 @@
 @import '~patternfly/dist/sass/patternfly/toolbar';
 @import '~patternfly/dist/sass/patternfly/blank-slate';
 @import '~patternfly/dist/sass/patternfly/list-view';
-@import '~patternfly/dist/sass/patternfly/popovers';
```
Comment (spadgett) [inline frontend/public/vendor.scss]: Looks like this might still be used in kubevirt-web-ui-components

cc @mareklibra @vojtechszocs

**Comment by rhamilto** [comment]
@matthewcarleton, should the icon and icon colors change for this instance, too?

**Comment by matthewcarleton** [comment]
yes @rhamilto I'd say so. @mceledonia can you confirm?

**Comment by rhamilto** [comment]
/hold

Should use https://github.com/openshift/console/pull/2733

**Comment by rhamilto** [comment]
/hold cancel

https://github.com/openshift/console/pull/2733 is held up by unit test failures.  Figure we should go ahead and get this in and circle back later.

@spadgett, PTAL.

**Comment by spadgett** [comment]
Are these references in kubevirt-web-ui-components a problem if we remove the PF3 partials?

```
❯ rg OverlayTrigger
src/components/Form/Dropdown.js
5:import { ButtonGroup, DropdownButton, MenuItem, noop, Tooltip, OverlayTrigger } from 'patternfly-react';
32:              <OverlayTrigger key={key} overlay={tooltip} placement="top">
34:              </OverlayTrigger>

src/components/Details/VmDetails/VmDetails.js
5:import { Button, Alert, FieldLevelHelp, Icon, Tooltip, OverlayTrigger } from 'patternfly-react';
232:            <OverlayTrigger key="template" overlay={tooltip} placement="top">
234:            </OverlayTrigger>

src/components/Details/VmTemplateDetails/VmTemplateDetails.js
5:import { Row, Col, Button, Alert, Icon, Tooltip, OverlayTrigger } from 'patternfly-react';
185:            <OverlayTrigger key="template" overlay={tooltip} placement="top">
187:            </OverlayTrigger>

src/components/Dashboard/TopConsumers/ConsumerItem.js
3:import { Row, Col, OverlayTrigger, ProgressBar, Tooltip } from 'patternfly-react';
11:          <OverlayTrigger
18:          </OverlayTrigger>
34:          <OverlayTrigger
41:          </OverlayTrigger>
```

**Comment by rhamilto** [comment]
> Are these references in kubevirt-web-ui-components a problem if we remove the PF3 partials?

I don't believe Popover is in use in https://github.com/kubevirt/web-ui-components.  But if it is, the kubevirt team can add it or any other necessary Bootstrap 3/PatternFly 3 partials back in https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/style.scss.  cc:  @mareklibra, @vojtechszocs

**Comment by mceledonia** [comment]
@matthewcarleton @rhamilto Yeah I'd say let's change it here too

**Comment by rhamilto** [comment]
> @matthewcarleton @rhamilto Yeah I'd say let's change it here too

D'oh.  This merged before I saw @mceledonia comment.  Will have to fix in a follow on.  

Note:  this component does not appear to be in use.  I didn't want to remove it since it's part of the dashboard owned by the CNV team.  @rawagner, can this component be removed?

**Comment by rhamilto** [comment]
This component being `DashboardCardHelp`.

**Comment by rawagner** [comment]
@rhamilto `DashboardCardHelp` is used in Ceph and Noobaa dashboards..or do you mean remove it and repace by `FieldLevelHelp` from https://github.com/openshift/console/pull/2733 ? If yes, then Im ok with that.

**Pull request merged**